drivers/pds/lib/utils.go

```diff
@@ -1181,8 +1199,61 @@ func CreateIndependentMySqlApp(ns string, podName string, appImage string, pvcNa
 	return pod, podName, nil
 }
 
-// CreatecassandraWorkload generate workloads on the cassandra db
-func CreatecassandraWorkload(cassCommand string, deploymentName string, namespace string) (*v1.Deployment, error) {
+//CreatePodWorkloads generate workloads as standalone pods
+func CreatePodWorkloads(name string, image string, creds WorkloadGenerationParams, namespace string, count string, env []string) (*corev1.Pod, error) {
+	var value []string
+	podSpec := &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: name + "-",
+			Namespace:    namespace,
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{
+					Name:  name,
+					Image: image,
+					Env:   make([]corev1.EnvVar, 4),
+				},
+			},
+			RestartPolicy: corev1.RestartPolicyOnFailure,
+		},
+	}
+
+	value = append(value, creds.Host)
+	value = append(value, creds.User)
+	value = append(value, creds.Password)
+	value = append(value, count)
+
+	for index := range env {
+		podSpec.Spec.Containers[0].Env[index].Name = env[index]
+		podSpec.Spec.Containers[0].Env[index].Value = value[index]
+	}
+
+	pod, err := k8sCore.CreatePod(podSpec)
+	if err != nil {
+		log.Errorf("An Error Occured while creating %v", err)
```

Comment:
just return the err and log it at the calling function

Reply:
Done